pyanaconda/bootloader/base.py

```diff
@@ -826,20 +800,27 @@ def set_boot_args(self, *args, **kwargs):
 
             self.boot_args.add(new_arg)
 
-        # passed-in objects
-        for cfg_obj in chain(args, kwargs.values()):
-            if hasattr(cfg_obj, "dracutSetupArgs"):
-                setup_args = cfg_obj.dracutSetupArgs()
-                self.boot_args.update(setup_args)
-                self.dracut_args.update(setup_args)
-            elif hasattr(cfg_obj, "dracut_setup_args"):
-                setup_args = cfg_obj.dracut_setup_args()
-                self.boot_args.update(setup_args)
-                self.dracut_args.update(setup_args)
-            else:
-                setup_string = cfg_obj.dracutSetupString()
-                self.boot_args.add(setup_string)
-                self.dracut_args.add(setup_string)
+    def _set_graphical_boot_args(self):
+        """Set up the graphical boot."""
+        args = []
+
+        try:
+            import rpm
+        except ImportError:
```

Comment:
I wonder if logging a warning there might make sense ? Not sure how serious the situation is though & if it warrants a warning.

Reply:
Fixed.